var/spack/repos/builtin/packages/py-tensorflow/1-1_fcc_tf_patch.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tensorflow/tensorflow.bzl b/tensorflow/tensorflow.bzl
```

Comment:
I assume this patch was introduced by accident?

Reply:
I added '1-1_fcc_tf_patch.patch' by mistake.
I deleted this file.